pylint/checkers/utils.py

```diff
@@ -849,9 +849,12 @@ def uninferable_final_decorators(
     for decorator in getattr(node, "nodes", []):
         if isinstance(decorator, nodes.Attribute):
             try:
-                import_node = decorator.expr.lookup(decorator.expr.name)[1][0]
+                _, import_nodes = decorator.expr.lookup(decorator.expr.name)
             except AttributeError:
                 continue
+            if not import_nodes:
```

Comment:
Could you add a comment about why this is the case? Just so we don't wonder about it later on.

Reply:
Makes sense.